packages/payment-detection/src/provider.ts

```diff
@@ -1,21 +1,34 @@
-import { ethers } from 'ethers';
+import { ethers, providers } from 'ethers';
 
-let configuration = {
+let defaultProviderOptions: Record<string, string> = {
   // fallback to Ethers v4 default projectId
   infura: process.env.RN_INFURA_KEY || '7d0d81d0919f4f05b9ab6634be01ee73',
 };
 
-export const initPaymentDetectionProvider = (config: Partial<typeof configuration>) => {
-  configuration = { ...configuration, ...config };
+let blockchainRpcs: Record<string, string> = {
+  matic: 'https://rpc-mainnet.matic.network/',
```

Comment:
missing other network like xDai & private ?

Reply:
private, done. 
xdai & others will be added when relevant. This is only used for payment networks FYI